packages/babel-generator/src/printer.ts

```diff
@@ -1128,6 +1128,7 @@ class Printer {
             if (
               this._buf.hasContent() &&
               (comment.type === "CommentLine" ||
+                // eslint-disable-next-line eqeqeq
                 commentStartLine != commentEndLine)
```

Comment:
Why do we need `!=` here?

Reply:
I realize that `null/undefined` shouldn't exist here.